src/trio/_core/_tests/test_run.py

```diff
@@ -2547,14 +2575,10 @@ async def main() -> NoReturn:
 
 async def test_nursery_strict_exception_groups() -> None:
     """Test that strict exception groups can be enabled on a per-nursery basis."""
```

Comment:
TODO: parametrize/change/duplicate these and related ones for `strict_exception_groups=False`